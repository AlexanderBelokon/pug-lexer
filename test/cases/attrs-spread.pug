div(...attrs)
div(...attrs=foo)
