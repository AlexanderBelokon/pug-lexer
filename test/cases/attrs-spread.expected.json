{"type":"tag","line":1,"col":1,"val":"div"}
{"type":"start-attributes","line":1,"col":4}
{"type":"spread-attribute","line":1,"col":5,"val":"attrs","mustEscape":true}
{"type":"end-attributes","line":1,"col":13}
{"type":"newline","line":2,"col":1}
{"type":"tag","line":2,"col":1,"val":"div"}
{"type":"start-attributes","line":2,"col":4}
{"type":"attribute","line":2,"col":5,"name":"a","val":"'bleh'","mustEscape":true}
{"type":"spread-attribute","line":2,"col":15,"val":"attrs","mustEscape":true}
{"type":"end-attributes","line":2,"col":23}
{"type":"newline","line":3,"col":1}
{"type":"tag","line":3,"col":1,"val":"div"}
{"type":"start-attributes","line":3,"col":4}
{"type":"attribute","line":3,"col":5,"name":"a","val":"'bleh'","mustEscape":true}
{"type":"spread-attribute","line":3,"col":14,"val":"attrs","mustEscape":true}
{"type":"end-attributes","line":3,"col":22}
{"type":"newline","line":4,"col":1}
{"type":"tag","line":4,"col":1,"val":"div"}
{"type":"start-attributes","line":4,"col":4}
{"type":"attribute","line":4,"col":5,"name":"a","val":true,"mustEscape":true}
{"type":"spread-attribute","line":4,"col":7,"val":"attrs","mustEscape":true}
{"type":"end-attributes","line":4,"col":15}
{"type":"newline","line":5,"col":1}
{"type":"eos","line":5,"col":1}